packages/suite/src/views/wallet/anonymize/components/CoinjoinConfirmation.tsx

```diff
@@ -117,16 +118,17 @@ export const CoinjoinConfirmation = ({ account }: CoinjoinConfirmationProps) =>
     const targetAnonymity = useSelector(selectCurrentTargetAnonymity);
     const roundsNeeded = useSelector(state => selectRoundsNeeded(state, account.key));
     const roundsFailRateBuffer = useSelector(selectRoundsFailRateBuffer);
+    const defaultMaxMiningFee = useSelector(state =>
+        selectDefaultMaxMiningFeeByAccountKey(state, account.key),
+    );
 
     const dispatch = useDispatch();
 
     const { coinjoinSessionBlockedMessage, isCoinjoinSessionBlocked } = useCoinjoinSessionBlockers(
         account.key,
     );
 
-    const anonymitySet = account.addresses?.anonymitySet;
-
-    if (!coinjoinClient || !targetAnonymity || !anonymitySet || !coinjoinAccount) {
```

Comment:
why this change?

Reply:
I noticed that `anonymitySet` was used in this component in the past, but not anymore.

Reply:
@dahaca is it ok to remove this condition from here?